mixins/demo/visibility-test-list.js

```diff
@@ -0,0 +1,73 @@
+import '../../components/button/button.js';
+import './visibility-test-item.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+import { VisibilityMixin } from '../visibility-mixin.js';
+
+export class VisibilityTestList extends VisibilityMixin(LitElement) {
+
+	static get properties() {
+		return {
+			_changeTracker: { type: Number, reflect: false },
```

Comment:
Is this used somewhere? It seems like it's only being assigned to, but not assigned to anything else.

Reply:
I'm using it to trigger a re-render whenever an item is added or removed. I'll switch it to just manually call `requestUpdate()` to make that clearer.